backend/backend/celery.py

```diff
@@ -4,7 +4,8 @@
 
 from celery import Celery
 from django.conf import settings
-from utils.constants import ExecutionLogConstants
+
+# from utils.constants import ExecutionLogConstants
```

Comment:
Please remove the imports if not getting used.

Reply:
@athul-rs this is causing unit test cases to break with circular import error. Let me know your thoughts

![Screenshot from 2024-05-28 10-53-41](https://github.com/Zipstack/unstract/assets/110175055/a18f9241-6804-475b-9294-1e7c57559ce9)

Reply:
cc: @muhammad-ali-e